packages/aws-cdk-lib/aws-ecs-patterns/lib/base/network-multiple-target-groups-service-base.ts

```diff
@@ -35,7 +35,7 @@ export interface NetworkMultipleTargetGroupsServiceBaseProps {
    *
    * @default - none
    */
-  readonly taskImageOptions?: NetworkLoadBalancedTaskImageProps;
+  readonly taskImageOptions?: NetworkLoadBalancedFargateTaskImageProps;
```

Comment:
You shouldn't need to update this since the base class doesn't need the fargate inputs.

Reply:
Ok so I need not update the taskImageOptions since the base class doesn't need the fargate inputs.
So I updated the code by add following interface in fargate-service-base.ts 
and we can import that interface to the fargate patterns and use ephemeral storage in it 
```ts 
export interface FargateTaskImageOptions {
  readonly ephemeralStorageGiB?: number;
}
```
Changes you can see in the latest commit